src/Payum/Core/Resources/docs/get-it-started.md

```diff
@@ -13,13 +13,15 @@ The preferred way to install the library is using [composer](http://getcomposer.
 Run composer require to add dependencies to _composer.json_:
 
 ```bash
-php composer.phar require payum/offline
+php composer.phar require payum/offline php-http/guzzle6-adapter
```

Comment:
`php-http/guzzle6-adapter` has to be added to all gateway's get-it-started docs.